src/standalone/Selector/BaseSelector.tsx

```diff
@@ -75,7 +76,7 @@ export interface BaseSelectorProps<DataT extends BaseSelectorData>
 	 * Data load function
 	 * @param search The user search input
```

Comment:
missing docs for param switchValue

Reply:
@TQjona , Changes updated. Please Review.